main.js

```diff
@@ -195,21 +199,19 @@
                     zIndexOffset: 100
                 });
                 // Probably a better way to format the library popup but its cleaner for now
-                record.pin.bindPopup(`
-                <strong>${record.Name}</strong>
-                <br> Address: ${record.Address}
-                <br> Phone: ${record.Phone}
-                <br> Monday: ${record.MoOpen.substring(0, 5)} - ${record.MoClose.substring(0, 5)}
-                <br> Tuesday: ${record.TuOpen.substring(0, 5)} - ${record.TuClose.substring(0, 5)}
-                <br> Wednesday: ${record.WeOpen.substring(0, 5)} - ${record.WeClose.substring(0, 5)}
-                <br> Thursday: ${record.ThOpen.substring(0, 5)} - ${record.ThClose.substring(0, 5)}
-                <br> Friday: ${record.FrOpen.substring(0, 5)} - ${record.FrClose.substring(0, 5)}
-                <br> Saturday: ${record.SaOpen.substring(0, 5)} - ${record.SaClose.substring(0, 5)}
-                <br> Sunday: ${record.SuOpen.substring(0, 5)} - ${record.SuClose.substring(0, 5)}
-                `);
+                record.pin.bindPopup(JSONtoTable(record));
                 record.pin.addTo(map);
                 markers.push(record);
             });
         });
+    function JSONtoTable(result){
+        var table = "<table>"
+        for(var key in result){
+            if(key == "_full_text" || key == "pin") continue;
+                table += `<tr><td> ${key}</td><td>${result[key]}</td></tr>`
```

Comment:
You're right, that is definitely not indented properly. I updated that as well as a few of the other things you suggested. Thanks!